src/index.html

```diff
@@ -13,5 +13,111 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    
+    <fieldset>
+      <legend>Personal information</legend>
+      
+      <div class="form-field">
```

Comment:
```suggestion
      <label class="form-field">
```
add styles to `.form-field` selector instead of using `div`